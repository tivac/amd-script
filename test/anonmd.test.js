"use strict";

describe("anonmd", () => {
    it("Should load a single anonymous module", async () => {
        await page.goto(require.resolve("./specimens/simple.html"));

        await expect(page.evaluate(() => document.body.innerHTML)).resolves.toMatchSnapshot();
    });
});
