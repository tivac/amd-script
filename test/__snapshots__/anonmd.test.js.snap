// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`amd-script Should load a single anonymous module 1`] = `

<div>
  ./simple.js
</div>

`;

exports[`amd-script Should support module dependencies 1`] = `

<div>
  ./dependencies-b.js
</div>
<div>
  ./dependencies-c.js
</div>
<div>
  ./dependencies-a.js
</div>

`;
